### PR TITLE
add bwpayer table and actions

### PR DESCRIPTION
**Pull request by danielvo11**
## Change Description

- Add table to store bandwidth payer
- Create action to set and accept bandwidth payer

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: src/orng.cpp [hunk 1]
```diff
@@ -70,6 +71,41 @@ ACTION orng::version() {
...
+            rec.accepted = false;
+        });
+    }
+}
+
+ACTION orng::acceptbwpay(const eosio::name& payee, const eosio::name& payer, bool accepted) {
+    check(!is_paused(), "Contract is paused");
+    require_auth(payer);
+
+    auto it = bwpayers_table.find(payee.value);
```
Comment (porkchop) [inline src/orng.cpp]: I think we can shorten this to `require_find`, and the text below should probably be `payee does not exist`

**Review by porkchop — CHANGES_REQUESTED** [review]
Looks great! just a minor nitpick to adjust

File: src/orng.cpp (hunk 1 above)
Comment (danielvo11) [reply src/orng.cpp]: Resolved

**Review by porkchop — APPROVED** [review]
LGTM!
